schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+    id INTEGER PRIMARY KEY,
```

Comment:
[OPTIONAL] Nice job making the id a primary key :+1: It would be better if you could also make it not null and auto incremented: 
```
    id int NOT NULL IDENTITY(1,1)
```